compiler/luci/pass/src/QuantizeDequantizeWeightsPass.cpp

```diff
@@ -180,6 +207,102 @@ void cal_minmax_per_channel(CircleConst *node, std::vector<float> &min, std::vec
   }
 }
 
+void sym_wquant_per_channel(CircleConst *node, std::vector<float> &min, std::vector<float> &max,
+                            std::vector<float> &scaling_factor, std::vector<int64_t> &zp,
+                            std::vector<float> &nudged_min, std::vector<float> &nudged_max)
+{
+  assert(node->dtype() == loco::DataType::FLOAT32);
+  const int32_t kMinScale = std::numeric_limits<int16_t>::lowest() + 1;
+  const int32_t kMaxScale = std::numeric_limits<int16_t>::max();
+
+  uint32_t size = node->size<loco::DataType::FLOAT32>();
+  std::vector<int32_t> quantized_values(size);
+
+  for (size_t i = 0; i < min.size(); ++i)
+  {
+    compute_sym_scale_zp(min[i], max[i], scaling_factor[i], zp[i], nudged_min[i], nudged_max[i]);
+  }
+
+  loco::TensorShape dimension;
+  dimension.rank(4);
+  uint32_t indices[4] = {
+      0,
+  };
+  int channel_dim_index{0};
+
+  bool ret = get_channel_dim_index(node, dimension, channel_dim_index);
+  assert(ret);
```

Comment:
`ret` will fail as unused var in release build. please fix this as previous fix I've made.